README.rst

```diff
@@ -1,114 +1,218 @@
+* `Examples`_
 * `Contributing`_
+* `Community`_
+* `Citation`_
 
-Overview
---------
+Ivy as a transpiler
+-------------------
+
+Ivy's transpiler allow you to use code from any other framework (and soon, from any other version of the same framework!) in your own code with just one line of code. To do so, Ivy traces a computational graph and leverages Ivy's frontends and backends to link one framework to the other. 
+
+This way, Ivy makes all ML-related projects available for you, independently of the framework you want to use to research, develop, or deploy systems. Feel free to head over to the docs for the full API reference, but the main functions that you'd use probably are:
+
+.. code-block:: python
+
+    ivy.compile()     # Compiles a function into an efficient graph, removing Ivy's wrapping and redundant code
+    ivy.transpile()   # Converts framework specific code to a different framework
+    ivy.unify()       # Converts framework specific code to Ivy
+
+These functions can be used eagerly and lazily. If you pass the neccesary arguments, the function will be called instantly, otherwise, compilation/transpilation will happen the first time you invoke the function with the proper arguments.
+
+.. code-block:: python
+    
+    import ivy
+    ivy.set_backend("jax")
+
+    # Simple JAX function to transpile
+    def test_fn(x):
+        return jax.numpy.sum(x)
+
+    x1 = ivy.array([1., 2.])
 
-Ivy is an ML framework that currently supports JAX, TensorFlow, PyTorch, and Numpy.
-We’re very excited for you to try it out!
+.. code-block:: python
+    
+    # Arguments are available -> transpilation happens eagerly
+    eager_graph = ivy.transpile(test_fn, to="torch", args=(x1,))
+    
+    # eager_graph is now torch code and runs efficiently
+    ret = eager_graph(x1)
+
+.. code-block:: python
+    
+    # Arguments are not available -> transpilation happens lazily
+    lazy_graph = ivy.transpile(test_fn, to="torch")
+    
+    # The transpiled graph is initialized, transpilation will happen here
+    ret = lazy_graph(x1)
+    
+    # lazy_graph is now torch code and runs efficiently
+    ret = lazy_graph(x1)
+
+If you want to learn more, you can find more information in the `Ivy as a transpiler section of the docs!<https://lets-unify.ai/ivy/design/ivy_as_a_transpiler.html>`_
+
+When should I use Ivy as a transpiler?
+######################################
+
+If you want to use building blocks published in other frameworks (neural networks, layers, array computing libraries, training pipelines...), you want to integrate code developed in various frameworks, or maybe straight up move code from one framework to another, the transpiler is definitely the tool 🔧 for the job! As the output of transpilation is native code in the target framework, you can use the converted code just as if it was code originally developed in that framework, appliying framework-specific optimizations or tools, making a whole new level of code available to you.
```

Comment:
"making a whole new level of code available to you" -> "instantly exposing your project to all of the unique perks of a different framework."